packages/altair-koa-middleware/package.json

```diff
@@ -15,15 +15,15 @@
     "@types/supertest": "^2.0.11",
     "jest": "29.4.1",
     "koa": "^2.13.1",
+    "koa-send": "^5.0.1",
     "nodemon": "^2.0.12",
     "supertest": "^6.1.6",
     "ts-jest": "29.0.5",
     "ts-node": "^10.2.1",
     "typescript": "4.9.5"
   },
   "peerDependencies": {
-    "@koa/router": "~11.0.2",
-    "koa-send": "^5.0.1"
```

Comment:
These were semver major changes, but they were published in non semver major version changes.

Reply:
Sorry I'm not sure I follow. You mean I should have published a major version since the change is major?

Reply:
> You mean I should have published a major version since the change is major?

Yes.

Anytime a package makes its peer dependency requirements tighter (require newer versions), it's a breaking change that should be published in a semver major version increment.

Packages that were previously using `@koa/router` v11 and `altair-koa-middleware` will now have npm install errors.

Reply:
Got it. I can publish a major version next but I don't think that would fix the issue with those packages, as they would need to update manually.

Reply:
I'm a bit rusty about the correct way to fix an accidentally major change in a minor release, but maybe what you could do is revert the major change in an other patch release so that way people using `altair-koa-middleware` v5 are fixed (for projects not using a lock file , which are most open source published packages, CI will just start working for them again, and npm install for them will start working again), and then redo the major changes correctly in a follow-up v6 release.

Reply:
Sounds good